backend/siarnaq/api/compete/models.py

```diff
@@ -335,6 +332,14 @@ def try_rating_update(self):
                 opponents=[o for o in participants if o is not participant]
             )
 
+    def report_to_bracket(self, is_private):
```

Comment:
Make `is_private` kwarg-only

```suggestion
    def report_to_bracket(self, *, is_private):
```